src/components/zen-tab/test/zen-tab.spec.tsx

```diff
@@ -10,4 +11,36 @@ describe('zen-tab', () => {
 
     expect(page.root.shadowRoot).toBeTruthy();
   });
+
+  it('should display a badge', async () => {
+    const page = await newSpecPage({
+      components: [ZenTab],
+      html: `<zen-tab badge="123">Tab 1</zen-tab>`,
+    });
+
+    expect(page.root.shadowRoot.querySelector('.badge')).toBeTruthy();
+  });
+
+  it('should be disabled', async () => {
+    const page = await newSpecPage({
+      components: [ZenTab],
+      html: `<zen-tab disabled>Tab 1</zen-tab>`,
+    });
+
+    expect(page.root.hasAttribute('disabled')).toBeTruthy();
+  });
+
+  it('should trigger on click event', async () => {
+    const page = await newSpecPage({
+      components: [ZenTab],
+      html: `<zen-tab>Tab 1</zen-tab>`,
+    });
+
+    const clickSpy = jest.fn();
+    page.root.addEventListener('click', clickSpy);
+    simulateMouse('click', page.root);
+    await page.waitForChanges();
+
+    expect(clickSpy).toHaveBeenCalled();
```

Comment:
Not a blocker, but I would suggest you to use `toHaveBeenCalledTimes` over `toHaveBeenCalled` as it prevents undesired side effects (like the double `focus` we had on the date picker).

```suggestion
    expect(clickSpy).toHaveBeenCalledTimes(1);
```

Reply:
Done